styles/go.dev/catppuccin.user.css

```
@@ -107,7 +107,8 @@
     --pink: @red;
     --turq-dark: @accent-color;
     --white: @text;
-
+    --border-code: 0.0625rem @surface0 solid;
```

Better to follow `--color-border` and use `surface2` for uniformity.
```suggestion
    --border-code: 0.0625rem @surface2 solid;
```